Titanium.Web.Proxy/Network/CertificateManager.cs

```diff
@@ -219,6 +220,51 @@ public void TrustRootCertificate()
         }
 
         /// <summary>
+        /// Puts the certificate to the local machine's certificate store. 
+        /// Needs elevated permission. Works only on Windows.
+        /// </summary>
+        /// <returns></returns>
+        public bool TrustRootCertificateAsAdministrator()
+        {
+            if (RunTime.IsRunningOnMono())
+            {
+                return false;
+            }
+
+            var fileName = Path.GetTempFileName();
+            File.WriteAllBytes(fileName, RootCertificate.Export(X509ContentType.Pkcs12));
+
+            var info = new ProcessStartInfo
+            {
+                FileName = "certutil.exe",
+                Arguments = "-importPFX -Enterprise -p \"\" -f \"" + fileName + "\"",
```

Comment:
Isn't Local Machine is what we need instead of Enterprise? I think if we don't specify -Enterprise it installs for local machine.